docs/design-proposals/2021-11-12-validation-webhook-for-CRs.md

```diff
@@ -0,0 +1,68 @@
+# Validation webhook for CRs
+
+## Summary
+
+This document presents a design to validate and admit webhook for CRs, which contains:
+
+- TidbCluster
+- TidbMonitor
+- TidbInitializer
+- Backup
+- Restore
+- BackupSchedule
+- TidbClusterAutoScaler
+
+### Goals
```

Comment:
Webhook can be run in `controller-manager` process just like operator built by [kubebuilder](https://book.kubebuilder.io/cronjob-tutorial/webhook-implementation.html). Could you add it into your proposal because old webhook component maybe deprecated and no need to deploy.